x-pack/plugins/actions/README.md

```diff
@@ -26,15 +26,19 @@ Table of Contents
     - [Executor](#executor)
     - [Example](#example)
   - [RESTful API](#restful-api)
-    - [`POST /api/actions/action`: Create action](#post-apiaction-create-action)
-    - [`DELETE /api/actions/action/{id}`: Delete action](#delete-apiactionid-delete-action)
-    - [`GET /api/actions`: Get all actions](#get-apiactiongetall-get-all-actions)
-    - [`GET /api/actions/action/{id}`: Get action](#get-apiactionid-get-action)
-    - [`GET /api/actions/list_action_types`: List action types](#get-apiactiontypes-list-action-types)
-    - [`PUT /api/actions/action/{id}`: Update action](#put-apiactionid-update-action)
-    - [`POST /api/actions/action/{id}/_execute`: Execute action](#post-apiactionidexecute-execute-action)
+    - [`POST /api/actions/action`: Create action](#post-apiactionsaction-create-action)
```

Comment:
is this correct? looks funny

Reply:
This is created from an automated markdown tool.